jenkins/common.groovy

```diff
@@ -45,8 +45,7 @@ def isReviewable() {
 def isDeployable() {
   return (IS_DEV_BRANCH ||
           IS_STAGING_BRANCH) &&
-    !env.CHANGE_TARGET &&
-    !currentBuild.nextBuild // if there's a later build on this job (branch), don't deploy
+    !env.CHANGE_TARGET
```

Comment:
> CHANGE_TARGET
For a multibranch project corresponding to some kind of change request, this will be set to the target or base branch to which the change could be merged, if supported; else unset.

Seems okay to leave this.

Reply:
Is https://ci.eclipse.org/webtools/env-vars.html/ the source where you found the definition for `CHANGE_TARGET`?

Reply:
Yep! Lol